grapple/types/pages.py

```diff
@@ -117,28 +117,41 @@ class Meta:
         interfaces = (PageInterface,)
 
 
-def get_specific_page(id, slug, token, content_type=None):
+def get_specific_page(id=None, slug=None, url=None, token=None, content_type=None):
```

Comment:
is there a plan to use this outside `PagesQuery.Mixin.resolve_page` ? if not, move it there?

Reply:
It’s used in the subscription code, at the bottom of the file (can’t link; I’m on mobile)